web-local/pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>com.bazaarvoice.emodb</groupId>
         <artifactId>emodb-parent</artifactId>
-        <version>6.0.8-SNAPSHOT</version>
+        <version>6.0.10-SNAPSHOT</version>
```

Comment:
Any idea why git is showing this? master in on 6.0.10-SNAPSHOT already....

Reply:
uh...not sure.. maybe it just needs a rebase? idk... I checked this file out from master. Maybe the change is duplicated as a result or something.